### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 # Imgproxy
+
 [![Build Status](https://github.com/bmuller/imgproxy/actions/workflows/ci.yml/badge.svg)](https://github.com/bmuller/imgproxy/actions/workflows/ci.yml)
-[![Hex pm](http://img.shields.io/hexpm/v/imgproxy.svg?style=flat)](https://hex.pm/packages/imgproxy)
-[![API Docs](https://img.shields.io/badge/api-docs-lightgreen.svg?style=flat)](https://hexdocs.pm/imgproxy/)
+[![Module Version](https://img.shields.io/hexpm/v/imgproxy.svg)](https://hex.pm/packages/imgproxy)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/imgproxy/)
+[![Total Download](https://img.shields.io/hexpm/dt/imgproxy.svg)](https://hex.pm/packages/imgproxy)
+[![License](https://img.shields.io/hexpm/l/imgproxy.svg)](https://github.com/bmuller/imgproxy/blob/master/LICENSE.md)
+[![Last Updated](https://img.shields.io/github/last-commit/bmuller/imgproxy.svg)](https://github.com/bmuller/imgproxy/commits/master)
```
Comment (bmuller) [inline README.md]: I'm not sure these "badges" are all that informative.  Stylistically, I'd like to omit.
```suggestion
```

File: mix.exs [hunk 2]
```diff
@@ -12,7 +12,6 @@ defmodule ImgProxy.MixProject do
       elixir: "~> 1.8",
       start_permanent: Mix.env() == :prod,
       deps: deps(),
-      description: "imgproxy URL generator and helper functions",
```
Comment (bmuller) [inline mix.exs]: Was this removal intentional?

File: mix.exs [hunk 3]
```diff
@@ -43,29 +46,28 @@ defmodule ImgProxy.MixProject do
...
       extra_applications: [:logger]
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:credo, "~> 1.5", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.25", only: :dev}
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false}
```
Comment (bmuller) [inline mix.exs]: What's the reasoning for relaxing the version requirement?  I don't think I'd want to convey that _any_ version of `ex_doc` would work.

**Comment by bmuller** [comment]
Thanks for the fixes!  Overall, looks great - just have a few questions / comments.

File: README.md (hunk 1 above)
Comment (kianmeng) [reply README.md]: Noted. Removed.

Comment (kianmeng) [reply README.md]: Noted. Removed.

File: mix.exs (hunk 2 above)
Comment (kianmeng) [reply mix.exs]: It's moved under `package/0`, which have the same effect, but more relevant.

File: mix.exs (hunk 3 above)
Comment (kianmeng) [reply mix.exs]: We want to get the latest version of `ex_doc`.

**Comment by kianmeng** [comment]
@bmuller Sorry, I accidentally created the PR based on the `master` branch. I've opened another PR based on feature branch instead. See https://github.com/bmuller/imgproxy/pull/2.
